src/affexpr.jl

```diff
@@ -83,10 +102,12 @@ end
 # Alias for (Float64, Variable), the specific GenericAffExpr used by JuMP
 const AffExpr = GenericAffExpr{Float64,Variable}
 AffExpr() = zero(AffExpr)
+AffExpr(v::Variable) = AffExpr([v], [1.], 0.)
```

Comment:
Why is it needed to define these constructors ?

Reply:
This particular one is a little weird, I'm not 100% sure why this one was throwing the warning.  It must have something to do with the alias.  Of course, the reason the constructors were added in general is because `convert` now falls back to constructors, and not vice versa.

Reply:
Apparently the constructor is used, because I'm getting the warning, but it's very difficult to track down exactly what is causing it.

Reply:
Ok then it's good :)